susemanager-utils/cluster-providers/caasp/formulas/caasp-management-node-formula/caasp-management-node-formula.spec

```diff
@@ -0,0 +1,54 @@
+#
+# spec file for package caasp-management-node-formula
+#
+# Copyright (c) 2020 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+%define fname caasp-management-node
+%define fdir  %{_datadir}/salt-formulas
+Name:           caasp-management-node-formula
+Version:        0.1
+Release:        0
+Summary:        Salt formula for configuring a CaaSP management node
+License:        GPL-2.0-only
+Group:          Applications/System
+Source:         %{name}-%{version}.tar.gz
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+BuildArch:      noarch
+
+%description
+Salt formula for configuring a CaaS Platform management node.
+
+%prep
+%setup -q
+
+%build
+
+%install
+mkdir -p %{buildroot}%{fdir}/states/%{fname}
+mkdir -p %{buildroot}%{fdir}/metadata/%{fname}
+cp -R caasp-management-node/* %{buildroot}%{fdir}/states/%{fname}
+cp -R metadata/* %{buildroot}%{fdir}/metadata/%{fname}
+
+%files
+%defattr(-,root,root)
+%dir %{_datadir}/salt-formulas
```

Comment:
You already have `fdir`. Why this is needed?

Also... don't we have other package owning this path? I am pretty sure there is, so this could provoke a conflict and we should instead depend on the other package. Check a head server.

Reply:
Afaik a directory can be owned by multiple packages. Please see the section about "Common directory for unrelated packages" [here](https://en.opensuse.org/openSUSE:Specfile_guidelines#Ownership).

Reply:
I also checked on the ref server. `/usr/share/susemanager` is owned by various packages.

Reply:
@mateiw right, I checked the doc.

IMHO this is sometimes masking missing dependencies. But OK, let's keep it.